wolf/system/test/postgresql.hpp

```diff
@@ -0,0 +1,217 @@
+#if defined(WOLF_TEST) && defined(WOLF_SYSTEM_POSTGRESQL)
+
+#pragma once
+
+#include <wolf.hpp>
+#include <system/w_leak_detector.hpp>
+#include <boost/test/included/unit_test.hpp>
+
+#include <system/db/w_postgresql.hpp>
+
+using wolf::system::pgdb::w_pgdb_conection;
+using wolf::system::pgdb::w_pgdb_execresult;
+using wolf::system::pgdb::w_pgdb_queryresult;
+using wolf::system::pgdb::w_pgdb_record;
+
+#define POSTGRESQL_SERVER_ADDRESS "172.26.218.210"
```

Comment:
A real accessible test server is required here.

this wrapper library does not provide additional functionality but reforms the design into a more developer friendly db driver library, upon libpq. thus for this reason, along with dependency injection overhead cost, the libpq usage is not mocked.

p.s. it would've been better if it was in examples instead of tests.

Reply:
it'll be refactored later by code owner or someone else.